src/plugins/tables/tables-fr.hbs

```diff
@@ -978,6 +978,131 @@
 			</tr>
 		</tbody>
 	</table>
+
+	<p>Ajouter la pagination au-dessus du tableau en le gardant en dessous.</p>
```

Comment:
Please add a subheading to have a clearer separation between both working example.

```suggestion
	<h4>Exemple avec une pagination en haut du tableau</h4>
	<p>Ajouter la pagination au-dessus du tableau en le gardant en dessous en utilisant la configuration suivante: <code>data-wb-tables='{"ordering" : false, "dom":"&lt;\u0027top\u0027pfil>rt&lt;\u0027bottom\u0027p>&lt;\u0027clear\u0027>"}'</code></p>
```